tests/plain-nodaily/test.toml

```
@@ -0,0 +1,56 @@
+# Test graphite_index with disable-daily-index = false
+[test]
+config_template = "carbon-clickhouse.conf.tpl"
+input = [ 
+    "test.host1.cpu.loadavg 10.2 1625478240",
+    "test.host1.cpu.loadavg 9.4 1625478300",
+]
+
+[[test.clickhouse]]
+version = "19.7"
+dir = "tests/clickhouse/rollup"
+
+[[test.clickhouse]]
+version = "20.4"
+dir = "tests/clickhouse/rollup"
+
+[[test.clickhouse]]
+version = "20.8"
+dir = "tests/clickhouse/rollup"
+
+[[test.clickhouse]]
+version = "21.3"
+dir = "tests/clickhouse/rollup"
+
+[[test.clickhouse]]
+version = "21.6"
```

IMHO, it makes sense to use `latest`, since it points to the most recent stable release in the docker hub
```suggestion
version = "latest"
```